pkg/collector/python/datadog_agent.go

```diff
@@ -59,6 +59,13 @@ func GetClusterName(clusterName **C.char) {
 	*clusterName = TrackedCString(goClusterName)
 }
 
+// GetTracemallocEnabled exposes the tracemalloc configuration of the agent to Python checks.
+//export GetTracemallocEnabled
+func GetTracemallocEnabled() C.bool {
+	trace := config.TracemallocEnabled(config.Datadog)
```

Comment:
Following my other comments this would become `return C.bool(config.Datadog.GetBool("tracemalloc_debug"))`